### ISSUE TITLE
VCARD not displaying correctly in Chatwoot

### ISSUE THREAD

**Issue by dlc-letelier** [EvolutionAPI/evolution-api#253]
Hello, 

there is an issue and when a contact VCARD is send to Chatwoot, it only appears the name:

<img width="398" alt="image" src="https://github.com/EvolutionAPI/evolution-api/assets/60855725/188e64ea-29b9-456c-910c-770044e13804">

Here is a log of the evolution-api showing the error:

![image](https://github.com/EvolutionAPI/evolution-api/assets/60855725/4d394464-a1a6-4ba0-9527-992d8f1ff84a)
<img width="515" alt="image" src="https://github.com/EvolutionAPI/evolution-api/assets/60855725/d2811825-4d61-42d7-b35b-9d3732ddda50">

I try to check in the code, and made some test on the code:

```
var value = {
      vcard: 'BEGIN:VCARD\n' +
        'VERSION:3.0\n' +
        'N:;Cesar;;;\n' +
        'FN:Cesar\n' +
        'TEL;type=CELL;type=VOICE;waid=18092244067:+1 (809) 224-4067\n' +
        'TEL;type=CELL;type=VOICE;waid=18092242067:+2 (809) 224-4067\n' +
        'END:VCARD'
};

const vCardData = value.vcard.split('\n');
const contactInfo = {};

vCardData.forEach((line) => {
  const [key, value] = line.split(':');
    if (key && value) {
      contactInfo[key] = value;
    }
  });

  let formattedContact = `**Contact:**\n**name:** ${contactInfo['FN']}\n`;


  let numberCount = 1;
    Object.keys(contactInfo).forEach((key) => {
      if (key.includes('TEL')) {
        const phoneNumber = contactInfo[key];
        formattedContact += `**number ${numberCount}:** ${phoneNumber}\n`;
        numberCount++;
      }
  });

console.log(formattedContact)
```

And now I am getting the correct formating:
<img width="398" alt="image" src="https://github.com/EvolutionAPI/evolution-api/assets/60855725/c0a8ad41-9608-4b53-b37e-204f4b87419a">

Basically, I removed the "key.startsWith('item')" in line 1203 and 1231

https://github.com/EvolutionAPI/evolution-api/blob/8fe75cd2101cf71277280b46276208e47ff78f5a/src/whatsapp/services/chatwoot.service.ts#L1231C15-L1231C37

Maybe, this will help to find the solution

**Comment by DavidsonGomes** [comment]
Excelente, muito obrigado pelos logs informados, vamos corrigir isso ja!

**Comment by DavidsonGomes** [comment]
Corrigido na versao 1.6.0
